modules/core/02-client/abci_test.go

```diff
@@ -74,3 +77,55 @@ func (suite *ClientTestSuite) TestBeginBlockerConsensusState() {
 	suite.Require().NoError(err)
 	suite.Require().Equal(bz, consState)
 }
+
+func (suite *ClientTestSuite) TestBeginBlockerWithUpgradePlan_EmitsUpgradeChainEvent() {
```

Comment:
nit: I don't think underscores in test names are convention in Go, right? 
Could use `TestBeginBlockerUpgradeEvents` and `TestBeginBlockerUpgradeEventsAbsence`?

But I mean its probably fine to be honest, the names are meaningful as is!

Reply:
yeah no harm simplifying the name a bi, great suggestion!